src/styles/blocks/card.scss

```diff
@@ -0,0 +1,148 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: $card-width;
+  height: 408px;
```

Comment:
Don't fix container size (if there is no such a requirement)
Let the content size dictate it. To avoid overflow or accidental scroll bar
```suggestion
```